internals/secrethub/run.go

```diff
@@ -90,18 +101,65 @@ func (cmd *RunCommand) Run() error {
 	}
 	envSources = append(envSources, flagSource)
 
-	if cmd.template != "" {
-		file, err := ioutil.ReadFile(cmd.template)
+	if cmd.envFile == "" {
+		const defaultEnvFile = "secrethub.env"
+		_, err := os.Stat(defaultEnvFile)
 		if err != nil {
-			return ErrCannotReadFile(cmd.template, err)
+			if !os.IsNotExist(err) {
+				return fmt.Errorf("could not read default run env-file %s: %s", defaultEnvFile, err)
+			}
+		} else {
+			cmd.envFile = defaultEnvFile
 		}
+	}
 
-		tplSource, err := NewEnvTemplate(string(file))
-		if err != nil {
-			return errio.Error(err)
+	osEnv, err := parseKeyValueStringsToMap(os.Environ())
+	if err != nil {
+		return errio.Error(err)
+	}
+
+	templateVars := make(map[string]string)
+
+	for k, v := range osEnv {
```

Comment:
Should we do this if `template_version=1`? It makes that the command does not run if any `SECRETHUB_VAR` variable is set.

Reply:
We discsussed this in person and concluded that this behaviour is undesirable, but because it is an edge case of a feature that is decprecated, we will leave it as is. Changing it requires too many changes.